src/components/Form/RadiobuttonGroup.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlSpan } from '../../reset';
+import { VisuallyHidden } from '../Visually-hidden/Visually-hidden';
+import { idGenerator } from '../../utils/uuid';
+
+import { RadiobuttonProps } from './Radiobutton';
+
+const baseClassName = 'fi-radiobuttongroup';
+const radiobuttonGroupClassNames = {
+  container: `${baseClassName}_container`,
+  label: `${baseClassName}_label`,
+  hintText: `${baseClassName}_hintText`,
+};
+
+type Label = 'hidden' | 'visible';
+
+export interface RadiobuttonGroupProps {
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Radiobutton or RadiobuttonDivider */
+  children: Array<React.ReactElement<RadiobuttonProps>>;
+  /**
+   * Hint text to be displayed under the label.
+   */
+  hintText?: string;
+  /** Label for group */
+  label: string;
+  /** Hide or show label. Label element is always present, but can be visually hidden.
+   * @default visible
+   */
+  labelMode?: Label;
+  /**
+   * Unique id
+   * If no id is specified, one will be generated using uuid
+   * @default uuidV4
+   */
+  id?: string;
+  /** Name for the group; this will be set to Radiobuttons */
+  name: string;
+  /** Value that is selected from group, should match with the value given to Radiobutton */
+  value?: string;
```

Comment:
`value` is basically required for radiobutton to work correctly at least in the uses cases that I thought, but it is optional for now. We may want to change this if needed.

Reply:
I'd put it as required then. I see no use cases where you want a radiobutton without value, especially if it is required for it to function correctly.

Reply:
I noticed that my original comment for this belongs actually to **Radiobutton** instead of **RadiobuttonGroup**.
Here the value should be optional as it acts like controlling the selected radio if using this.

Reply:
Ah, meant my comment for the radiobutton as well.